src/components/secrets/Secret.tsx

```diff
@@ -256,6 +456,12 @@ export function ListComponent({ icon = '', title, subtitle = '', onClick, classN
         </article>
     )
 }
+
+interface EsoData {
```

Comment:
extract out interfaces to some types file

Reply:
Done